redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/TreehouseAppContent.kt

```diff
@@ -453,6 +460,13 @@ private class ViewContentCodeBinding<A : AppService>(
         )
       }
     }
+
+    bindingScope.launch {
+      while (true) {
+        delay(5.seconds)
+        leakDetector.checkLeaks()
+      }
+    }
```

Comment:
This is still in the wrong place. It should be owned by the creator of the `LeakDetector`. I'll probably add an API like `leakDetector.checkIn(scope, every = 5.seconds)` or something.

Reply:
Would it be more efficient to do it on-demand? Such as after calls to `sendChanges` that removed things?

Reply:
GC runs now only occur when a reference is being watched